### DISCUSSION TITLE
Plausible dump data to others for further processing

### DISCUSSION THREAD

**Discussion by prk2331** [plausible/analytics#2843] [Self-Hosted Support]
Hi team
    Does the Plausible have any native extensions or native method capability to dump the data to kafka for further processing whether in self host or oncloud  ?
        By data i mean (Visitor Id, revisit or not , device name, screen size, sessionId etc.)

Thanks.

**Comment by abh** [comment]
With self-hosting you could, I think, setup a table in Clickhouse that's writing to Kafka and then a materialized view to "tee off" writes to the events and sessions tables (or events_v2 / sessions_v2) into the "Kafka table".
